test/modules/encoders/test_clip_text_encoder.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from test.test_utils import assert_expected
+from torchmultimodal.modules.encoders.clip_text_encoder import CLIPTextEncoder
+from torchmultimodal.utils.common import get_current_device
+
+torch.manual_seed(1234)
+context_length = 77
+device = get_current_device()
+
+
+class TestCLIPTextEncoder:
+    def test_clip_init(self):
```

Comment:
I'm not sure that this test is really testing the CLIP initialization explicitly. Maybe you can add checks on some of the parameters that get set in `initialize_parameters`.

Reply:
added a check for some layers that the mean of the weights is 0